woocommerce-gateway-ebanx/includes/class-wc-ebanx-oxxo-gateway.php

```diff
@@ -89,14 +90,18 @@ public static function thankyou_page($order)
 		$oxxo_print = $oxxo_url . "&format=print";
 		$customer_email = get_post_meta($order->id, '_ebanx_payment_customer_email', true);
 		$oxxo_hash = get_post_meta($order->id, '_ebanx_payment_hash', true);
+		$customer_name = get_post_meta($order->id, '_billing_first_name', true);
```

Comment:
If you have access to the `$order` object, you can use the magic property `billing_first_name`, and you can replace to this: `$customer_name = $order->billing_first_name;` 👍